client/src/components/CustomCountBadge/CustomCountBadge.tsx

```diff
@@ -0,0 +1,30 @@
+import { Badge, BadgeProps } from 'antd';
+import React from 'react';
+
+type PresetColors = BadgeProps['status'];
+
+function getStylesByStatus(status?: PresetColors) {
+  // Add other status if needed
+  switch (status) {
+    case 'default':
+      return {
+        backgroundColor: '#f0f2f5',
+        color: 'rgba(0, 0, 0, 0.45)',
+      };
+    case 'processing':
+      return {
+        backgroundColor: '#e6f7ff',
+        color: '#1890ff',
+      };
+    default:
+      return {};
+  }
+}
+
+function CustomCountBadge(props: BadgeProps) {
+  const { status, ...restProps } = props;
+
+  return <Badge {...restProps} style={getStylesByStatus(status)} />;
+}
```

Comment:
I would reconsider creating such component:
1. the name is too vague, why custom, what is the business.
2. seems it might be more efficient to use standard component with right props

Reply:
1. Got it. Will change. Maybe just CountBadge will be ok
2. Standard components doesn't support status prop with count. Should be passed only one of them. According to designs count badge should have colors in antd approach, but in fact antd provides only red count. So, I decided to "extend" it and apply styles by status prop. Seems like it's not very clear 🙂 I will think about it